Cargo.toml

```diff
@@ -43,8 +44,9 @@ getrandom = { version = "0.2", features = ["js"], optional = true }
 [dev-dependencies]
 hex = "0.4"
 json = "0.12"
-sha2 = "0.9"
+proptest = "1"
 regex = "1"
+sha2 = "0.9"
 voprf = { path = "", default-features = false, features = ["std"] }
 
 [package.metadata.docs.rs]
```

Comment:
I am assuming this isn't something we want to show up on docs.rs?

Reply:
Can you clarify this comment, or perhaps you meant to highlight a different line? I did not quite understand...

Reply:
Note: in the docs it would show up with a note anyway saying it requires the feature "danger" to access.